src/dmd/compiler.d

```diff
@@ -12,7 +12,59 @@
 
 module dmd.compiler;
 
+///
 struct Compiler
 {
-    const(char)* vendor; // Compiler backend name
+    /// Compiler backend name
+    const(char)* vendor;
 }
+
+/**
+ * Initializes the dmd lexer and all required components in order
+ *
+ * It is templated (so D only) to prevent
+ *  errors on local imports to things that may not exist.
+ *
+ * See_Also:
+ *     dmdLexerDeinit
+ */
+void dmdLexerInit()()
```

Comment:
This function is already in a package called `dmd`.  Is it necessary to have the `dmd` prefix?